core/src/main/scala/com/emaginalabs/kuronometer/core/model/BuildExecutionReport.scala

```diff
@@ -0,0 +1,26 @@
+package com.emaginalabs.kuronometer.core.model
+
+case class ProjectInformation(name: Option[String], description: Option[String])
+
+case class ProjectVersion(group: Option[String], version: String)
+
+case class ProjectDefinition(info: ProjectInformation, version: ProjectVersion)
```

Comment:
Review the json parsing. It has been updated since the first time we reviewed the implementation. All the information needed can be found [here](https://github.com/pedrovgs/Kuronometer/blob/master/kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/domain/BuildExecution.scala).